genome.spec

```diff
@@ -181,36 +195,65 @@ typedef structure {
 typedef int Bool;
 
 /*
-Genome object holds much of the data relevant for a genome in KBase
-    Genome publications should be papers about the genome
-Should the Genome object contain a list of contig_ids too?
-Source: allowed entries RefSeq, Ensembl, Phytozome, RAST, Prokka, User_upload
-    #allowed entries RefSeq, Ensembl, Phytozome, RAST, Prokka,
-User_upload controlled vocabulary managed by API
-
-Domain is a controlled vocabulary
-Warnings : mostly controlled vocab but also allow for unstructured
-Genome_tiers : controlled vocabulary (based on ap input and API checked)
-Allowed values: #Representative, Reference, ExternalDB, User
-
-Examples Tiers:
-All phytozome - Representative and ExternalDB
-Phytozome flagship genomes - Reference, Representative and ExternalDB
-Ensembl - Representative and ExternalDB
-RefSeq Reference - Reference, Representative and ExternalDB
-RefSeq Representative - Representative and ExternalDB
-RefSeq Latest or All Assemblies folder - ExternalDB
-User Data - User tagged
-
-Example Sources:
-RefSeq, Ensembl, Phytozome, Microcosm, User, RAST, Prokka, (other annotators)
-
+Field descriptions:
+    id: string - object id
+    scientific_name: string - human readable species name
+    domain: string - human readable phylogenetic domain name
+    warnings: list of string - genome-level warnings generated in the annotation process
+    genome_tiers: list of string - controlled vocabulary (based on app input and checked by the GFU)
+        Allowed values: Representative, Reference, ExternalDB, User
+        Tier assignments based on genome source:
+         * All phytozome - Representative and ExternalDB
+         * Phytozome flagship genomes - Reference, Representative and ExternalDB
+         * Ensembl - Representative and ExternalDB
+         * RefSeq Reference - Reference, Representative and ExternalDB
+         * RefSeq Representative - Representative and ExternalDB
+         * RefSeq Latest or All Assemblies folder - ExternalDB
+         * User Data - User tagged
+    feature_counts: map of string to integer - total counts of each type of feature
+        (key examples: "gene", "CDS", "mRNA", "non_coding_features")
+    genetic_code: int - An NCBI-assigned taxonomic category for the organism
+        See here: https://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi 
+    dna_size: integer - total number of nucleotides
+    num_contigs: integer - total number of contigs in the genome
+    molecule_type: string - the type of molecule sequenced (eg. "DNA")
```

Comment:
Are there any other types supported? For a genome it's always going to be DNA, yeah?

Reply:
I assume so, will ask Jason